packages/kotti-ui/source/kotti-field-select/types.ts

```diff
@@ -32,7 +32,13 @@ export namespace Shared {
 }
 
 export namespace KottiFieldSingleSelect {
-	export type Props = KottiField.Props<Value> & Shared.Props
+	export type Props = KottiField.Props<Value> &
+		Shared.Props & { actions: KottiFieldSingleSelect.Action[] }
+
+	export type Action = {
+		label: string
```

Comment:
your type definition doesn't match the template usage of `action.value`

Reply:
the use of action.value was a mistake from me. action doesn't contain value